docs/yaml/03-deployment.yaml

```diff
@@ -23,7 +23,7 @@ spec:
           imagePullPolicy: Always
           command: ["gangway", "-config", "/gangway/gangway.yaml"]
           env:
-            - name: GANGWAY_SESSION_SECURITY_KEY
+            - name: SESSION_SECURITY_KEY
```

Comment:
Prefixing the variable name with the app name is correct (see [envconfig](https://github.com/kelseyhightower/envconfig), i.e. no change is needed otherwise the test would've failed: https://github.com/heptiolabs/gangway/blob/793afd5699384d05d800a8a43710d5677666a2f7/cmd/gangway/config_test.go#L37

Reply:
@ch-dimitri  is correct, there should be no change here.

Reply:
Hmm, I received an error earlier around this environment variable and thought this was in error but you're correct. I was not aware of how `envconfig` worked. I'll close this PR.